includes/wpmenucart-woocommerce.php

```diff
@@ -91,5 +91,10 @@ public function get_cart_total() {
 
 			return $cart_contents_total;
 		}
+
+		public function get_cart_url() {
+			$wc_cart_url = apply_filters( 'woocommerce_get_cart_url', wc_get_page_permalink( 'cart' ) );
```

Comment:
Maybe we should add a comment here on why we are not using the previous function anymore.

Reply:
Added a comment. Please let me know what you think.

Reply:
Perfect!